core/src/main/java/org/bitcoinj/wallet/Wallet.java

```diff
@@ -4471,40 +4509,6 @@ public boolean canSignFor(Script script) {
         return candidates;
     }
 
-    /** Returns the {@link CoinSelector} object which controls which outputs can be spent by this wallet. */
-    public CoinSelector getCoinSelector() {
```

Comment:
I might leave this method or make it `getDefaultCoinSelector` if you take my advice on leaving it in a `protected final` field.

Reply:
I'm leaving it in, and changed the JavaDoc a bit.